tests/TestPassphraseGenerator.cpp

```diff
@@ -49,6 +49,6 @@ void TestPassphraseGenerator::testWordCase()
 
     generator.setWordCase(PassphraseGenerator::TITLECASE);
     passphrase = generator.generatePassphrase();
-    QRegularExpression regex("^([A-Z][a-z]* ?)+$");
-    QVERIFY(regex.match(passphrase).hasMatch());
+    QRegularExpression regex("^(?:[A-Z][a-z-]* )*[A-Z][a-z-]*$");
+    QVERIFY2(regex.match(passphrase).hasMatch(), passphrase.toStdString().c_str());
```

Comment:
I believe it is better to use the `qPrintableString` macro

Reply:
Thanks, didn't know about that! (Looks like it's called [`qPrintable`](https://doc.qt.io/qt-6/qstring.html#qPrintable).)